srv/wordpress/wp-content/plugins/acore-wp-plugin/src/hooks/user/user.php

```diff
@@ -343,14 +342,27 @@ function save_extra_user_profile_fields( $user_id ) {
      * @var \ACore\Account\Entity\AccountEntity
      */
     $gameUser = $accRepo->findOneByUsername($user->user_login);
-    if (!$gameUser)
-        throw new \Exception(__("Game account doesn't exist!","acore-wp-plugin"));
+    if (!$gameUser) {
+        if (isset($_POST['pass1']) && $_POST['pass1'] != '') {
+
+            create_account_if_not_exists($user, $_POST['pass1']);
```

Comment:
so, basically by adding this we make sure that if a user changes the password while updating the profile, the password check also will be executed?

Reply:
No, this is useful when:
- the user creates a wordpress account
- during the **game account** creation something goes wrong
- the user changes the password -> if the account does not exist, instead of throw an error it creates a new **game account** and solves the issue

I don't know why in the first step the game account has not been created but in this way we can solve the issue.